portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsDashboard.testcase

```diff
@@ -252,8 +257,13 @@ definition {
 	}
 
 	@description = "Bug: LRAC-12694 | Automation ID: LRAC-12811 | Test Summary: Known Individuals is not duplicated if user exists in multiple properties"
+	@ignore = "true"
 	@priority = 3
 	test IndividualNotDuplicatedIfUserExistsInMultipleProperties {
+
+		// AC Refactor: Data doesn't appear on time
+		// AC Refactor ticket: LRAC-13994
+
```

Comment:
Hey @joshitagaki, I see this test is passing constantly in the release routine, why should we disable it? I think we can continue with this case.

[History](https://testray.liferay.com/home/-/testray/case_results/2345384537/history?tab=result&hideFilterPin=true)

Reply:
Hmm this looks like very flaky behavior on upstream. I think we can ignore this on upstream only

Reply:
The two routines should have the same behavior. I think we can quarantine this case to see how it behaves.

I wanted to create a ticket so that we don't have tests skipped in the upstream and not in the release anymore. They are using the same emulator, so I think it's good for us to run all the tests on both to be able to verify these differences (it could be a bug in the master that we can catch before to backport to the UAT)

What do you think?

Reply:
Hmm yea quarantine might be better then